backend/pyproject.toml

```diff
@@ -12,6 +12,8 @@ uvicorn = {extras = ["standard"], version = "^0.15.0"}
 SQLAlchemy = "^1.4.29"
 asyncpg = "^0.25.0"
 alembic = "^1.7.5"
+python-jose = {extras = ["cryptography"], version = "^3.3.0"}
```

Comment:
why this package? Isn't pyjwt enough?

Reply:
https://github.com/mpdavis/python-jose/issues/67

Reply:
do we use the "at_hash"?

Reply:
dont think so.. i actually didnt know that pyjwt now supports a the jwk sets. (we do use those)
should we use pyjwt instead of jose?